GeneticAlgorithm.py

```diff
@@ -0,0 +1,301 @@
+#Import functions to be used
+import numpy as np #For array manipulations
+import random #for drawing random bits and setting seed
+import sys #Store the biggest number
+
+
+def create_Population(N_pop,n):
```

Comment:
I'd rename this to create_population, rather than create_Population. There are some other functions below that should also be renamed to have no capital letters.